src/components/Home.js

```diff
@@ -0,0 +1,53 @@
+import { useSelector, useDispatch } from 'react-redux';
+import React, { useEffect } from 'react';
+import {} from '../store/rootReducer';
+import { Link } from 'react-router-dom';
+import { fetchData } from '../store/actions';
+
+const Characters = () => {
+  const characterStore = useSelector((state) => state.characters.characters);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!characterStore.length) {
+      dispatch(fetchData());
+    }
+  }, []);
+
```

Comment:
- [ ] As it is a requirement for this project, please kindly add  a `Filter stateless component` to filter data retrieved from the API using given parameter(`e.g character type`). See screenshot of the requirements below. 

![filter](https://user-images.githubusercontent.com/65852621/144339263-9f0d211c-b9da-4f59-915a-fa9da00f257e.png)